Update SearchService README (#808)
NuGet/NuGet.Services.Metadata: src/NuGet.Services.SearchService/README.md

ignoreFilter | boolean | `true` to include unlisted packages and ignore the `prerelease` parameter
countOnly    | boolean | `true` to return only the total count and no metadata
sortBy       | string  | Sort results using a specified ordering

> packageType  | string  | Filter results to only show the ones with the specified package type

Reviewer: What do you think of this?

```suggestion
packageType  | string  | Filter results to those with the specified package type
```

- `title-desc` - sort by title, descending case-insensitive lexicographical order
- `created-asc` - sort by created timestamp, ascending chronological order
- `created-desc` - sort by created timestamp, descending chronological order
- `totalDownloads-asc` - sort by total downloads count, ascending numerical order
- `totalDownloads-desc` - sort by total downloads count, descending numerical order

The `relevance` value is used by default or if the provided value is not supported.

For `title-asc` and `title-desc`, a package's ID is used if the package has no explicit title value. Given that package
`title` is no longer prominently shown in NuGet experiences, this sorting order is only maintained for legacy reasons.

> The `packageType` parameter is a free form input. It defaults to an empty string, which means no filter. If there are no packages with the specified packageType in a request, an empty array will be returned.

Reviewer: I'd clarify which array you're referring:

```suggestion
The `packageType` parameter is a free form input. It defaults to an empty string, which means no filter. If there are no packages with the specified packageType in a request, an empty `data` array will be returned.
```
